snmp/datadog_checks/snmp/data/default_profiles/infoblox-ipam.yaml

```diff
@@ -0,0 +1,212 @@
+extends:
+  - _generic-host-resources-base.yaml
+  - _generic-if.yaml
+  - _generic-ucd.yaml
+  - infoblox.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.7779.1.1004    # Virtual Device
+  - 1.3.6.1.4.1.7779.1.1204    # IB-1552
+  - 1.3.6.1.4.1.7779.1.1301    # IB-550-A
+  - 1.3.6.1.4.1.7779.1.1302    # IB-1050-A
+  - 1.3.6.1.4.1.7779.1.1303    # IB-1550-A
+  - 1.3.6.1.4.1.7779.1.1401    # IB-810
+  - 1.3.6.1.4.1.7779.1.1402    # IB-820
+  - 1.3.6.1.4.1.7779.1.1403    # IB-1410
+  - 1.3.6.1.4.1.7779.1.1404    # IB-1420
+  - 1.3.6.1.4.1.7779.1.1405    # IB-1400
+  - 1.3.6.1.4.1.7779.1.1406    # IB-800
+  - 1.3.6.1.4.1.7779.1.1407    # PT-1400
+  - 1.3.6.1.4.1.7779.1.1411    # IB-2200
+  - 1.3.6.1.4.1.7779.1.1412    # IB-2210
+  - 1.3.6.1.4.1.7779.1.1413    # IB-2220
+  - 1.3.6.1.4.1.7779.1.1414    # PT-2200
+  - 1.3.6.1.4.1.7779.1.1421    # IB-4010
+  - 1.3.6.1.4.1.7779.1.1422    # IB-4030
+  - 1.3.6.1.4.1.7779.1.1423    # IB-4000
+  - 1.3.6.1.4.1.7779.1.1424    # PT-4000
+  - 1.3.6.1.4.1.7779.1.1425    # Infoblox, Model Unknown
+  - 1.3.6.1.4.1.7779.1.1431    # IB-100
+  - 1.3.6.1.4.1.7779.1.1441    # ND-800
+  - 1.3.6.1.4.1.7779.1.1442    # ND-1400
+  - 1.3.6.1.4.1.7779.1.1443    # ND-2200
+  - 1.3.6.1.4.1.7779.1.1444    # ND-4000
+  - 1.3.6.1.4.1.7779.1.1501    # IB-815
+  - 1.3.6.1.4.1.7779.1.1502    # IB-825
+  - 1.3.6.1.4.1.7779.1.1503    # IB-1415
+  - 1.3.6.1.4.1.7779.1.1504    # IB-1425
+  - 1.3.6.1.4.1.7779.1.1505    # IB-1405
+  - 1.3.6.1.4.1.7779.1.1506    # IB-805
+  - 1.3.6.1.4.1.7779.1.1507    # PT-1405
+  - 1.3.6.1.4.1.7779.1.1511    # IB-2205
+  - 1.3.6.1.4.1.7779.1.1512    # IB-2215
+  - 1.3.6.1.4.1.7779.1.1513    # IB-2225
+  - 1.3.6.1.4.1.7779.1.1514    # PT-2205
+  - 1.3.6.1.4.1.7779.1.1521    # IB-4015
+  - 1.3.6.1.4.1.7779.1.1522    # IB-4035
+  - 1.3.6.1.4.1.7779.1.1523    # IB-4005
+  - 1.3.6.1.4.1.7779.1.1524    # PT-4005
+  - 1.3.6.1.4.1.7779.1.1525    # IB-4025
+  - 1.3.6.1.4.1.7779.1.1541    # ND-805
+  - 1.3.6.1.4.1.7779.1.1542    # ND-1405
+  - 1.3.6.1.4.1.7779.1.1543    # ND-2205
+  - 1.3.6.1.4.1.7779.1.1544    # ND-4005
+  - 1.3.6.1.4.1.7779.2.2       # IB-1050
+  - 1.3.6.1.4.1.7779.2.4       # IB-1552
```

Comment:
do we have a source for this ?
I can't find any reference to `1.3.6.1.4.1.7779.2`, the MIBs only have `1.3.6.1.4.1.7779.1.*` or `1.3.6.1.4.1.7779.3.*`

Reply:
Looking at the infoblox docs they are present here https://docs.bmc.com/docs/discovery/contentref/infoblox-51-1073952259.html